static/css/main.css

```diff
@@ -345,24 +348,77 @@ div.divider {
   margin-bottom: 20px;
 }
 
-/* line 287, main.scss */
+/* line 290, main.scss */
 .vote_result {
   text-align: center;
   padding: 3px 5px;
   width: 80px;
 }
-/* line 292, main.scss */
+/* line 295, main.scss */
 .vote_result.vote_pass {
   color: #fff;
   background: #2bc77c;
 }
-/* line 297, main.scss */
+/* line 300, main.scss */
 .vote_result.vote_fail {
   color: #fff;
   background: #e64242;
 }
 
-/* line 304, main.scss */
+/* line 306, main.scss */
+.glossary {
+  margin-top: 40px;
+  padding: 20px;
```

Comment:
@tgrauer I had to comment out this line, and removed it from main.scss, as it made the glossary terms far too large for the table